src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from "react";
+import "./navbar.scsss";
```

Comment:
Ocorreu este bug para mim:

![image](https://user-images.githubusercontent.com/19687197/107990885-35a90d80-6fb4-11eb-97d0-e70d7b2303a6.png)

Acredito que o import esteja incorreto:

```suggestion
import "./navbar.scss";
```

Reply:
opa, hei de arrumar